pkg/reconciler/intevents/topic/topic_test.go

```diff
@@ -738,6 +797,12 @@ func TestAllCases(t *testing.T) {
 				}
 			}
 		}
+
+		// If we found "dataResidencyConfigMap" in OtherData, we create a store with the configmap
+		var drStore *dataresidency.Store
+		if cm, ok := testData["dataResidencyConfigMap"]; ok {
```

Comment:
It looks like it doesn't match the business logic. Can drStore be un-initialized? Wondering maybe for all test cases, the drStore should be a valid `*dataresidency.Store` except for error scenario of `*dataresidency.Store`

Reply:
The existing logic now allows `nil` value, in which case it will treat it as no special request for data residency.